comfy_api_nodes/nodes_moonvalley.py

```diff
@@ -688,19 +646,18 @@ async def generate(
 
         inference_params = MoonvalleyVideoToVideoInferenceParams(
             negative_prompt=negative_prompt,
-            seed=kwargs.get("seed"),
+            seed=seed,
             control_params=control_params,
         )
 
-        control = self.parseControlParameter(control_type)
+        control = parse_control_parameter(control_type)
 
         request = MoonvalleyVideoToVideoRequest(
             control_type=control,
             video_url=video_url,
             prompt_text=prompt,
             inference_params=inference_params,
         )
-        request.image_url = image_url if not image is None else None
```

Comment:
I removed this new input parameter from the node as it looks like an error introduced in this PR: 

https://github.com/comfyanonymous/ComfyUI/pull/9290

model `MoonvalleyVideoToVideoRequest` does not have `image_url` field. This line for me raises an exception during execution.

@guill would you mind sharing the reason this was added?

Reply:
because without these changes, the node is currently in a non-working state, I think that we are ok in the drop of the recently added **optional** `image` input that **broke** the node, and this is part of the fix.